Add Linux Mint 👻

Evernow opened the pull request:
Hacktober 🎃

Sources used: 

Mainly for release dates and support cycle: https://linuxmint.com/download_all.php


Repository explanation: https://forums.linuxmint.com/viewtopic.php?t=338979#:~:text=Linux%20Mint%2C%20being%20based%20on,their%20own%2C%20per%20the%20above.

32-bit support: https://blog.linuxmint.com/?p=3766

Evernow committed:
Add Linux Mint

captn3m0 commented on products/linuxmint.md:
```diff
@@ -0,0 +1,88 @@
+
+    
+---
+>[Linux Mint](https://linuxmint.com/) is a community-driven Linux distribution based on Ubuntu (which is then based on Debian), bundled with a variety of free and open-source applications. 
```
```suggestion
>[Linux Mint](https://linuxmint.com/) is a community-driven Linux distribution for desktop and laptop computers based on Debian and Ubuntu, bundled with a variety of free and open-source applications.
```

Changed wording to reflect upstream is both "Ubuntu and Debian" which is what the project seems to use:

>Linux Mint stands on the shoulder of giants, it is based on Debian and Ubuntu.

Both is important because https://www.linuxmint.com/download_lmde.php exists.

Content will need changes for LMDE, but should be straightforward.

Evernow replied:
Thanks for catching that! Completely missed it. I can look into that once I'm back home.

Evernow committed:
Redefine Linux Mint as Ubuntu and Debian based

Co-authored-by: Nemo <commits@captnemo.in>

Evernow committed:
Add information pertaining to LMDE, change release command

Changed command, lsd_release is not installed by default, changed command to show Linux Mint file referencing release.

captn3m0 commented on products/linuxmint.md:
```diff
@@ -0,0 +1,109 @@
+
+* Linux Mint has its own repositories for Linux Mint specific packages, but relies directly on Ubuntu repositories for the majority of the package base, including its mirrors. 
+* Linux Mint releases follows the support cycle as the Ubuntu release it is based on (which can be [referenced here](https://linuxmint.com/download_all.php), Linux Mint support officially does not extend to the duration of ESM. While Linux Mint users will receive [Extended Security Maintenance](https://ubuntu.com/security/esm) from Ubuntu once the main support period ends, Linux Mint officially c ...
+* Linux Mint enables the `Universe` Ubuntu repository by default, unlike Ubuntu which leaves this disabled. This is not recommended to be enabled as the [Ubuntu security team does not maintain this repository](https://wiki.ubuntu.com/SecurityTeam/FAQ#How_are_components_and_pockets_used_in_the_builds.2C_and_how_do_they_affect_security_updates.3F).
```
```suggestion
* Linux Mint enables the `Universe` Ubuntu repository by default, unlike Ubuntu which leaves this disabled as it is [maintained by the Ubuntu Community](https://wiki.ubuntu.com/SecurityTeam/FAQ#How_are_components_and_pockets_used_in_the_builds.2C_and_how_do_they_affect_security_updates.3F), and not the Ubuntu security team.
```

The implication here are that universe might not get timely fixes, but I'd prefer if we have a citation that can actually say if that is the case.

Evernow replied:
Added citation with https://github.com/endoflife-date/endoflife.date/pull/428/commits/02ab98840eed0ad480ed7671817697a63cf8345d

captn3m0 commented on products/linuxmint.md:
```diff
@@ -0,0 +1,109 @@
+* Linux Mint releases follows the support cycle as the Ubuntu release it is based on (which can be [referenced here](https://linuxmint.com/download_all.php), Linux Mint support officially does not extend to the duration of ESM. While Linux Mint users will receive [Extended Security Maintenance](https://ubuntu.com/security/esm) from Ubuntu once the main support period ends, Linux Mint officially c ...
+* Linux Mint enables the `Universe` Ubuntu repository by default, unlike Ubuntu which leaves this disabled. This is not recommended to be enabled as the [Ubuntu security team does not maintain this repository](https://wiki.ubuntu.com/SecurityTeam/FAQ#How_are_components_and_pockets_used_in_the_builds.2C_and_how_do_they_affect_security_updates.3F).
+
+## Relationship with Debian
```
Links are needed in this section.

Evernow replied:
Is this covered with https://github.com/endoflife-date/endoflife.date/commit/02ab98840eed0ad480ed7671817697a63cf8345d ?

captn3m0 commented on products/linuxmint.md:
```diff
@@ -0,0 +1,109 @@
+* Unlike Debian and Linux Mint which supports older releases past a new version coming out, LMDE stops supporting a release shortly after a new one comes out. This however is still subject to change on a release by release basis. There is no support period set in stone for any LMDE release.
+
+
+## Cadence
```
```suggestion
## Feature Support
```

Release Cadence usually means the expected release-cycle-timings, which this section isn't covering.

captn3m0 commented on products/linuxmint.md:
```diff
@@ -0,0 +1,109 @@
+## Relationship with Ubuntu 
+
+* Linux Mint has its own repositories for Linux Mint specific packages, but relies directly on Ubuntu repositories for the majority of the package base, including its mirrors. 
+* Linux Mint releases follows the support cycle as the Ubuntu release it is based on (which can be [referenced here](https://linuxmint.com/download_all.php), Linux Mint support officially does not extend to the duration of ESM. While Linux Mint users will receive [Extended Security Maintenance](https://ubuntu.com/security/esm) from Ubuntu once the main support period ends, Linux Mint officially c ...
```
- Make the opt-in to Ubuntu Advantage clear
- Clarify it's only "some packages" that ESM will cover.

```suggestion
* Linux Mint releases follows the support cycle as the Ubuntu release it is based on (which can be [referenced here](https://linuxmint.com/download_all.php), Linux Mint support does not extend to the duration of <abbr title="Extended Security Maintenance">ESM</abbr>. Linux Mint users can opt-in to receive [Extended Security Maintenance](https://ubuntu.com/security/esm) via [Ubuntu Advantage](https://ubuntu.com/advantage) for some packages once the main support period ends. But Linux Mint considers that release End of Life once it enters that state, so Linux Mint packages will not be receiving any updates.
```

captn3m0 reviewed (changes requested):
Haven't validated the dates, left a few comments.

Evernow committed:
Update products/linuxmint.md

Co-authored-by: Nemo <commits@captnemo.in>

Evernow committed:
Add source for Universe not receiving timely security updates

Evernow committed:
Cadence -> Feature Support

Co-authored-by: Nemo <commits@captnemo.in>

Evernow committed:
Update products/linuxmint.md

Co-authored-by: Nemo <commits@captnemo.in>

captn3m0 committed:
Move important information to top

Dropped LMDE section, it didn't have anything important other than the existence of LMDE.

captn3m0 committed:
Fix YAML indent

BiNZGi reviewed (approved):
I chcked the details and everything okay -> merge
